Fixes the following compile error:
CLASSPATH=../../..:./../../..:$CLASSPATH  -d ../../..  -classpath ./../../.. QtUtils.java (...) QXmlDeclHandlerInterface.java
/bin/sh: line 2: -d: command not found
make[6]: *** [classnoinst.stamp] Error 127
make[6]: Leaving directory `/var/tmp/fst/src/kdebindings-3.5.4/qtjava/javalib/org/kde/qt'
--- kdebindings-3.5.4/qtjava/javalib/org/kde/qt/Makefile.am.orig	2006-09-30 00:50:38.000000000 +0200
+++ kdebindings-3.5.4/qtjava/javalib/org/kde/qt/Makefile.am	2006-09-30 00:52:20.000000000 +0200
@@ -13,5 +13,5 @@
 JAVAROOT = ../../..
 CLASSPATH = $(srcdir)/../../..
 JAVACFLAGS = -classpath $(CLASSPATH)
-# Prevent stupid automake from putting JAVAC=javac in Makefile.in
-JAVAC = @JAVAC@
+JAVAC = gcj -C
+JAR = jar
--- kdebindings-3.5.4/qtjava/javalib/Makefile.am.orig	2006-09-30 02:35:56.000000000 +0200
+++ kdebindings-3.5.4/qtjava/javalib/Makefile.am	2006-09-30 02:36:14.000000000 +0200
@@ -20,6 +20,7 @@
 data_DATA = qtjava.jar
 datadir = $(kde_libraries)/java
  
+JAR = jar
 qtjava.jar: org/kde/qt/classnoinst.stamp
 	$(JAR) cf qtjava.jar org/kde/qt/*.class
 
--- kdebindings-3.5.4/kdejava/koala/org/kde/koala/Makefile.am.orig	2006-09-30 03:29:35.000000000 +0200
+++ kdebindings-3.5.4/kdejava/koala/org/kde/koala/Makefile.am	2006-09-30 03:32:20.000000000 +0200
@@ -12,6 +12,5 @@
 
 JAVAROOT = ../../..
 CLASSPATH = $(srcdir)/../../..:$(srcdir)/../../../../../qtjava/javalib/qtjava.jar:$(srcdir)
-JAVACFLAGS = -classpath $(CLASSPATH)
-# Prevent stupid automake from putting JAVAC=javac in Makefile.in
-JAVAC = @JAVAC@
+JAVACFLAGS = -classpath $(CLASSPATH) -encoding iso-8859-1
+JAVAC = gcj -C
--- kdebindings-3.5.4/kdejava/koala/Makefile.am.orig	2006-09-30 11:39:58.000000000 +0200
+++ kdebindings-3.5.4/kdejava/koala/Makefile.am	2006-09-30 11:40:28.000000000 +0200
@@ -27,6 +27,7 @@
 data_DATA = koala.jar
 datadir = $(kde_libraries)/java
  
+JAR = jar
 koala.jar: org/kde/koala/classnoinst.stamp
 	$(JAR) cf koala.jar org/kde/koala/*.class
 
